lib/lotus/commands/new.rb

```diff
@@ -27,6 +27,21 @@ def initialize(app_name, environment, cli)
       def start
         @command.start
       end
+
+      private
+      def _get_real_app_name(app_name)
+        case app_name
+        when '.'
+          ::File.basename(Dir.getwd)
+        when '/'
```

Comment:
@mengqing I like @jodosha suggestion on having `--path`. :+1: 

And yes, we could remove the `.` after having ^ implemented because it would error out due to permission denied anyway. Much thanks for taking care of this.